src/test/java/org/springframework/cloud/dataflow/acceptance/test/BatchRemotePartitionTests.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.acceptance.test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
+
+import org.junit.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.dataflow.acceptance.test.util.TestConfigurationProperties;
+import org.springframework.cloud.dataflow.core.ApplicationType;
+import org.springframework.cloud.dataflow.rest.resource.TaskExecutionResource;
+import org.springframework.cloud.deployer.spi.cloudfoundry.CloudFoundryConnectionProperties;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Executes acceptance tests for the batch remote partition task.
+ *
+ * @author David Turanski
+ */
+@EnableConfigurationProperties({ BatchRemotePartitionTests.CFConnectionProperties.class })
```

Comment:
When I try to run this locally I'm getting:
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'spring.cloud.deployer.cloudfoundry-org.springframework.cloud.dataflow.acceptance.test.BatchRemotePartitionTests$CFConnectionProperties': Could not bind properties to BatchRemotePartitionTests.CFConnectionProperties (prefix=spring.cloud.deployer.cloudfoundry, ignoreInvalidFields=false, ignoreUnknownFields=true, ignoreNestedProperties=false); nested exception is org.springframework.validation.BindException: org.springframework.boot.bind.RelaxedDataBinder$RelaxedBeanPropertyBindingResult: 5 errors
Field error in object 'spring.cloud.deployer.cloudfoundry' on field 'password': rejected value [null]; codes [NotNull.spring.cloud.deployer.cloudfoundry.password,NotNull.password,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [spring.cloud.deployer.cloudfoundry.password,password]; arguments []; default message [password]]; default message [may not be null]
Field error in object 'spring.cloud.deployer.cloudfoundry' on field 'username': rejected value [null]; codes [NotNull.spring.cloud.deployer.cloudfoundry.username,NotNull.username,NotNull.java.lang.String,NotNull]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [spring.cloud.deployer.cloudfoundry.username,username]; arguments []; default message [username]]; default message [may not be null]

Looks like were are requiring CFConnectionProperties.   I'm just running the test from my IDE but seems like that should work.